crates/sui-protocol-config/src/snapshots/sui_protocol_config__test__version_15.snap

```diff
@@ -21,6 +21,7 @@ feature_flags:
   narwhal_versioned_metadata: true
   zklogin_auth: true
   consensus_transaction_ordering: ByGasPrice
+  shared_object_deletion: true
```

Comment:
reflagging as the PR has been updated significantly. We will need to turn this off for  <= 20 (or if 20 has been shipped already anything <= 21)

Reply:
yes the entire protocol config cleanup and bump to 21 is coming shortly